modules/react/popup/lib/PopupCloseIcon.tsx

```diff
@@ -1,30 +1,29 @@
 import React from 'react';
 
 import {xIcon} from '@workday/canvas-system-icons-web';
-import {
-  styled,
-  createSubcomponent,
-  StyledType,
-  ExtractProps,
-} from '@workday/canvas-kit-react/common';
+import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
 import {TertiaryButton} from '@workday/canvas-kit-react/button';
+import {space} from '@workday/canvas-kit-react/tokens';
 
 import {usePopupCloseButton, usePopupModel} from './hooks';
 
-const closeIconSpacing = 4;
-
 export interface PopupCloseIconProps extends ExtractProps<typeof TertiaryButton, never> {}
 
-const StyledCloseIcon = styled(TertiaryButton)<StyledType & PopupCloseIconProps>({
-  position: 'absolute',
-  right: closeIconSpacing,
-  top: closeIconSpacing,
-});
-
```

Comment:
This was some clean up, these styles where overriding any style properties, so I moved these down below